### ISSUE TITLE
Safari Layout Broken

### ISSUE THREAD

**Issue by ajkblue** [mauriceoegerli/viewtube-vue#2174]
### Describe the Bug

When visiting the official viewtube instance (https://viewtube.io) or hosting my own instance, on the latest version of Safari (16.6) and on iOS 16.6 Viewtube is broken and displays videos twice on each page, but the second one usually works. See the attached screenshot below.

### Steps to Reproduce the Bug

1. Go to viewtube.io
2. Play any video (such as https://viewtube.io/watch?v=cFslUSyfZPc)
3. Top video loads forever and unplayable, second one towards the bottom works.

### Expected Behaviour

I expect the video to be able to play like normal

### Screenshot/Screen recording

![Screen Shot 2023-08-18 at 13 25 16](https://github.com/ViewTube/viewtube/assets/9651939/20cc0bc2-9a9e-4b6d-aedc-9d189af7ca2e)

### Device Info

- Device: Mac, iPhone 11
- OS: macOS 13, iOS 16.6
- Browser: Safari
- Version: 16.6

**Comment by moisout** [comment]
Would it be possible for you to send a screenshot of the developer console?

**Comment by ajkblue** [comment]
Yes, I don't see anything very interesting though that points to why it doesn't work:
![Screenshot 2023-08-23 at 6 52 07 PM](https://github.com/ViewTube/viewtube/assets/9651939/7be3ecc1-808b-4939-9489-0cf12dfed826)

**Comment by moisout** [comment]
@ajkblue Would you be able to test if this still occurs?
